src/dictionary/ui/search/Dictionary.test.tsx

```diff
@@ -29,14 +29,22 @@ describe('Searching for word', () => {
   })
 
   it('displays result on successfull query', async () => {
-    await renderDictionary('/dictionary?query=lemma')
+    await renderDictionary(
+      '/dictionary?word=lemma&meaning=some meaning&root=lmm&vowelClass=a/a'
```

Comment:
Maybe store the query in an object `{ word: 'lemma', ... }`, `stringify` it for the url and get the expected values from the object.